docs/deprecations.md

```diff
@@ -85,3 +85,9 @@ Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2
 ## awsKmsKeyArn references
 
 Plase use `provider.kmsKeyArn` and `functions[].kmsKeyArn`. `service.awsKmsKeyArn` and `functions[].awsKmsKeyArn` will be removed with v3.0.0
+
+<a name="CLOUDFRONT_CACHE_BEHAVIOR_FORWARDED_VALUES_AND_TTL"><div>&nbsp;</div></a>
```

Comment:
All deprecations should be added as first in a list (we keep them ordered from newest to oldest)